### PR TITLE
change log level to info

### PR DESCRIPTION
**Pull request by mogoodrich**
Let's set this back to the default, it's helpful when debugging...

### CODE REVIEW

**Comment by mogoodrich** [comment]
fyi @brandones

**Comment by brandones** [comment]
Is it? My experience with log-level:info is that it produces millions (literally) of trace-level logs from some AOP thing that logs each function entry and exit. This causes important information to very quickly be lost from the logs. I don't think there's any important information at the INFO level.

**Comment by mogoodrich** [comment]
@brandones  It should at least tell us what metadata sharing packages it's trying to install on startup... maybe I'm just used to the longer logs, but I found the logs from startup hard to follow yesterday because a lot of the "markers" I look for weren't there... we could potentially turn them back off after we get things working, but if the MDS package loading is still not working, I'd want them on if I'm going to debug further.

**Comment by brandones** [comment]
Right, MDS logs. Well, let's see how it goes. Maybe someone has finally fixed that AOP thing.
